website/docs/r/atracker_route.html.markdown

```diff
@@ -28,20 +28,24 @@ Review the argument reference that you can specify for your resource.
 
 * `name` - (Required, String) The name of the route. The name must be 1000 characters or less and cannot include any special characters other than `(space) - . _ :`.
   * Constraints: The maximum length is `1000` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9 -._:]+$/`
-* `receive_global_events` - (Required, Boolean) Indicates whether or not all global events should be forwarded to this region.
+* `receive_global_events` - **DEPRECATED** (Optional, Boolean) Indicates whether or not all global events should be forwarded to this region.  Use rules.locations instead with `global` included.
 * `rules` - (Required, List) Routing rules that will be evaluated in their order of the array.
 Nested scheme for **rules**:
-	* `target_ids` - (Required, List) The target ID List. Only 1 target id is supported.
+	* `target_ids` - (Required, List) The target ID List. All the events will be send to all targets listed in the rule. You can include targets from other regions.
+	* `locations` - (Required, List) Logs from these locations will be sent to the targets specified. Locations is a superset of regions including global and *.
 
 ## Attribute reference
 
 In addition to all argument references listed, you can access the following attribute references after your resource is created.
 
-* `id` - The unique identifier of the Activity Tracker Route.
```

Comment:
Generally all attributes shd not have (optional /Required) they are computed may be can you remove that for all attributes..something like below
created` - (String) The timestamp of the route creation time.
api_version` - (Integer) The API version of the route.

Reply:
same for all docs

Reply:
Done

Reply:
I still see the same 
`updated_at` - (Required, String) The timestamp of the route last updated time.

For computed we will not have either Required/Optional
it should be something like below
`updated_at` - (String) The timestamp of the route last updated time.